#Standard header
Header header

#######################
# Debug
#######################
# WORD_ENC1_PERIOD
int32 enc1_period
# WORD_ENC2_PERIOD
int32 enc2_period
# Debug registers
int32[20] debug
